mobile/src/androidTest/java/ch/epfl/sdp/mobile/screenshots/BitmapMatcher.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Hey @alexandrepiveteau, did we obtain a license to use this ?

Reply:
Based on my understanding, code licensed under Apache 2.0 can be re-used and distributed (according to the terms at https://www.apache.org/licenses/LICENSE-2.0), so we do not need to perform any particular steps to obtain the license.

Reply:
👍